api/configuration/logging_controller.go

```diff
@@ -0,0 +1,56 @@
+package configuration
+
+import (
+	"net/http"
+
+	"github.com/Peripli/service-manager/pkg/log"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/pkg/web"
+)
+
+// Controller logging configuration controller
+type Controller struct {
+}
+
+func (c *Controller) getLoggingConfiguration(r *web.Request) (*web.Response, error) {
+	ctx := r.Context()
+	logCfg := log.Configuration()
+	log.C(ctx).Debugf("Obtaining log configuration with level: %s and format: %s", logCfg.Format, logCfg.Level)
+
+	return util.NewJSONResponse(http.StatusOK, logCfg)
+}
+
+func (c *Controller) setLoggingConfiguration(r *web.Request) (*web.Response, error) {
+	ctx := r.Context()
+	loggingConfig := log.Configuration()
```

Comment:
This config should be validated before passing it to the `Configure` func

Reply:
BytesToObject does that - welcome to the hidden validation "lair"